com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -591,6 +591,8 @@ public HDRenderPipeline(HDRenderPipelineAsset asset, HDRenderPipelineAsset defau
             InitializeProbeVolumes();
             CustomPassUtils.Initialize();
 
+            m_ShadowManager.InitializeNonRenderGraphResources();
```

Comment:
Maybe put a comment here to explain why the "NonRenderGraph" stuff here is called no matter what.